### DISCUSSION TITLE
Realtime vc output is broken

### DISCUSSION THREAD

**Discussion by forerunrun** [voicepaw/so-vits-svc-fork#445] [Q&A]
when trying to run realtime inference on an rtx 3090 gpu (24GB) the output is completely inaudible and broken, is there a way to set the default gpu preference to use rtx 3090 gpu's? i've noticed that the gui defaults to gtx 1060, is this modifyable to be better compatible with later rtx gpu's?

**Comment by ainub** [comment]
Depending on the card number you want to use (nvidia-smi):
svc vc -m .. -c .. -d 'cuda:0'

**Comment by forerunrun** [comment]
thanks for the reply , so for instance i would use...
svc vc -m .. -c .. -d 'cuda:3090' ?
will this also run the gui so i can change audio io settings?

**Comment by sbersier** [comment]
1) ` -d cuda:X` (with X=0,1,2,...) specifies which GPU (GPU#0, GPU#1,...)  you want to use (in case you have more than one). In general, -d cuda:0 is the option you want to choose.
2) What OS? (linux, Mac, Windows) Is there no sound at all or is the sound messed up?
3) With `svcg` : In case you have multiple sound cards, you'll have to pick the right one in the menu (input device, output device).  
4) I have a RTX 3060 (with Ubuntu) and it works. So, I don't see why it wouldn't on a 3090.

**Comment by vertexgamer** [comment]
You need to speak loudly and clear for realtime to work, otherwise it misses words or don't start at all
